[Web] Allow underscore and hyphen in DKIM selector

Ashus opened the pull request:
This change allows to have cleaner DNS zones as mail security related entries usually begin with _.
Also the default dkim selector was changed to _dkim.
I tested it and Google recognizes this selector without a single hitch.

andryyy commented:
Please remove the changed default, we will merge then.

There is no need to make it `_dkim`. It is not a standard nor given by a RFC. I have actually never seen a DKIM like `_bla`.
Furthermore not all security related entries are `_` prefixed. What about SPF TXTs?

Please do not create a PR with a changed default without ever discussing it. :(

Commit:
[Web] Allow underscore and hyphen in DKIM selector

This change allows to have cleaner DNS zones as mail security related entries usually begin with _.

Ashus commented:
Ok, removed the default selector change, sorry for that.
Yes, SPF TXTs are usually without any third level domain, but _dmarc, _mta-sts, _smtp._tls, even _acme-challenge all begin with _.

andryyy commented:
Merci :)
